tests/rptest/services/admin.py

```diff
@@ -303,7 +306,11 @@ def is_leader_stable():
             f"can't get stable leader of {namespace}/{topic}/{partition} within {timeout_s} sec"
         )
 
-    def _request(self, verb, path, node=None, **kwargs):
+    def _request(self,
+                 verb: str,
+                 path: str,
+                 node: MaybeNode = None,
+                 **kwargs: Any):
```

Comment:
For `args` and `kwargs` we are mostly just stuck using `Any` since the args captured generally don't have any particular common type.